cluster/watchdog.go

```diff
@@ -107,7 +107,7 @@ func (w *Watchdog) rescheduleContainers(e *Engine) {
 			clusterNetworks := w.cluster.Networks().Uniq()
 			for networkName, endpoint := range c.Info.NetworkSettings.Networks {
 				net := clusterNetworks.Get(endpoint.NetworkID)
-				if net != nil && net.Scope == "global" {
+				if net != nil && (net.Scope == "global" || net.Scope == "swarm") {
```

Comment:
Why is `net.Scope == "swarm"` required here? Isn't this something we were discussing in another PR as well?